<launch>
  <remap from="openni_points" to="/planner_cloud"/>
  <remap from="map" to="/rtabmap/proj_map"/>
  <remap from="move_base_simple/goal" to="/planner_goal"/>
  
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find wpirm_navigation)/config/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find wpirm_navigation)/config/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find wpirm_navigation)/config/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find wpirm_navigation)/config/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find wpirm_navigation)/config/base_local_planner_params.yaml" command="load" />
  </node>
  <node pkg="nodelet" type="nodelet" name="obstacles_detection" args="load rtabmap_ros/obstacles_detection stereo_nodelet">
    <remap from="cloud" to="/zed/point_cloud/cloud_registered"/>
    <remap from="obstacles" to="/planner_cloud"/>

    <param name="frame_id" type="string" value="base_link"/>
    <param name="map_frame_id" type="string" value="map"/>
    <param name="min_cluster_size" type="int" value="20"/>
    <param name="max_obstacles_height" type="double" value="0.0"/>
  </node>

</launch>